docs/decisions/app/0004-uswds-in-react.md

```diff
@@ -0,0 +1,70 @@
+# U.S. Web Design System in React
+
+- Status: proposed
+- Deciders: [list everyone involved in the decision] <!-- optional -->
```

Comment:
👀 deciders 👀

Reply:
I'll update this with a list of whoever reviews this proposal

Reply:
I'm ok with this decision. WIC MT chose not to use the Truss library at the time because of accessibility concerns, but, unfortunately, I don't currently have an itemized list of those concerns and the engineer who did the assessment is on leave. Did you all do any accessibility assessments as part of your research?

Reply:
@rocketnova Interesting, I can take a closer look at this. My assumption is that ultimately it should be the USWDS HTML markup that's getting output, so am curious what a11y issues might be caused by the React version.

Reply:
I'm similarly okay with the change - it sounds like documenting accessibility trade-offs would be important to include in the ADR!

Reply:
Some notes from very quick digging:

- Found this [WIC draft](https://navasage.atlassian.net/wiki/spaces/MWDP/pages/463765530/Accessibility+in+NextJS+USWDS+Draft-+WIP), but not sure if it's what you were referring to @rocketnova. Nothing about react-uswds, so guessing probably not. Searched some of the WIC channels as well without much luck.
- A handful of [_closed_  a11y issues](https://github.com/trussworks/react-uswds/issues?q=is%3Aissue+label%3Aa11y+is%3Aclosed) but no known issues are currently open. 
- They have [some a11y linting in place](https://github.com/trussworks/react-uswds/blob/a0558b69ec5b99903cfa8edddf2d8b058f5e296c/.eslintrc#L54).
- Clicking through each of their Storybook stories and looking at the Accessibility panel didn't surface any a11y errors.
- Their component tests often query elements by test ID, rather than the accessibility tree, so no a11y coverage in their unit tests.
- The React components seem to output the HTML markup USWDS has in their documentation, although I didn't look at each one.

I'm hesitant to document there's cons with `react-uswds` a11y without a specific example, but I can mention that we haven't done a comprehensive a11y audit of its markup.

Reply:
Added some more content around this here, thoughts? https://github.com/navapbc/template-application-nextjs/pull/96/commits/0d77edabfa1d05bba23aadaf511763e2513f51a3

Reply:
@sawyerh Thanks for doing that pass looking into it. I feel good about the language you've included. I also wouldn't want to call out a11y issues without specific examples. 

I apologize for being too brief in my earlier comment and not clearly communicating: if I recall correctly, the a11y issues my engineer found were in USWDS html markup itself. One of the reasons we didn't go with the Truss library at the time was because they were pinned to an older version of USWDS and therefore make for a slower cycle of getting a11y improvements that might be made at the USWDS level.

Reply:
> One of the reasons we didn't go with the Truss library at the time was because they were pinned to an older version of USWDS and therefore make for a slower cycle of getting a11y improvements that might be made at the USWDS level.

@rocketnova Thanks for this additional context, that makes a lot of sense 👍🏻